server/common/search/deliverables.ts

```diff
@@ -0,0 +1,80 @@
+import { ParsedUrlQuery } from "querystring";
+import { Op, WhereOptions } from "sequelize";
+
+import models from "db/models";
+
+import { caseInsensitiveEqual } from "helpers/db/conditionals";
+import { getAssociation } from "helpers/db/models";
+import paginate, { calculateTotalPages } from "helpers/paginate";
+
+import { HttpBadRequestError } from "server/errors/http-errors";
+
+export default async function get(query: ParsedUrlQuery) {
+  const {
+    taskId,
+    creator,
+    search,
+    state,
+    networkName: marketplace,
+    networkChain: chain,
+    page = 1,
+    sortBy = "createdAt",
+    order = "DESC",
+  } = query;
+
+  const deliverableWhere: WhereOptions = {};
+  const userWhere: WhereOptions = {};
+  const marketplaceWhere: WhereOptions = {};
+  const chainWhere: WhereOptions = {};
+
+  if (creator)
```

Comment:
add && isAddress(creator) so we make sure the passed argument is what we expect it to be